simulation.py

```diff
@@ -63,9 +90,10 @@
         t.start()
     
     #create some send events    
-    for i in range(5):
+    for i in range(1):
```

Comment:
Does this create enough output for the video?

Reply:
Yes, because the only thing that is needed for this video is to have H1 and H2 send packets to H3 along different paths.